Common/Data/UniverseSelection/Universe.cs

```diff
@@ -29,6 +29,8 @@ namespace QuantConnect.Data.UniverseSelection
     /// </summary>
     public abstract class Universe : IDisposable
     {
+        private TimeSpan? _minimumTimeInUniverseRoundingInterval;
```

Comment:
I'll add a comment for him.
`What it aims to support`:
I'm using it to round the members `timeInUniverse`, why? **as a fix** for the fact that we can't guarantee the **exact** selection time, example `UniverseSettings.MinimumTimeInUniverse` could be 1 day and `timeInUniverse` 0.99 day. See related issue. 
Even if we used the new `StepTimeProvider` to set up a schedule, say advance time once a day, it seems error prone and weak to potential race conditions.

Made him nullable to avoid calling `AdjustMinimumTimeInUniverseRoundingInterval()` on each call to `CanRemoveMemeber()`, so I'm adding this field to hold the value and setting him just once. Can't use `UniverseSettings.MinimumTimeInUniverse` for rounding directly.

Let me know what you think :)

Reply:
Yup, this is what I figured it was used for -- I just want to make sure we capture the _why_ behind things as it will be incredibly helpful in the future. I think this approach makes sense given that variability of invocation time

👍